templates/default/testconnector-launch.sh.erb

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -e
+
+help() {
+    echo ""
+    echo "usage: $0 [input_json_filepath image_name]"
+    echo ""
+    exit 1
+}
+
+if [ "$#" != "2" ] || [ "$1" == "-h" ] ; then
+  help
+fi
+
+
+FILENAME=$(basename "$1")
+FOLDER=$(dirname "$1")
+IMAGE="$2"
+
+docker run --rm -v $FOLDER:/home $IMAGE /home/$FILENAME -v;
```

Comment:
I suppose you assume here that the image is already in the registry, then you would need to do similar to git
https://github.com/logicalclocks/hops-hadoop-chef/commit/3a8f812819b7aa8658df26383ade33ba1c9f59c6

Reply:
@DhananjayMukhedkar are you addressing this?

Reply:
ah i saw you got a pr there.

Reply:
yes, just trying to test this in a  new VM